bg_module.py

```diff
@@ -0,0 +1,23 @@
+import os
```

Comment:
This entire file should go inside a folder. Probably named something like 'module/bg_module.py'. Inside main.py, refer to this file as 'from module import bg_module.py'

Also, when the file is inside a folder named 'module', you can rename it just as 'bg.py' having said that 'bg.py' is not a good name!

Reply:
I have put the additional files into the module folder.
But I have retained the "module" in the name of the files, as there are many similar variables that will confuse the person who reads the code. It was also confusing to me.